services/app/src/components/player/bouncingDots/styles.css

```diff
@@ -0,0 +1,37 @@
+body {
```

Comment:
This applies style to body tag, which is global, do we really need this? If we do, it should be in some other place, not next to bouncingDots styles.